src/components/AnchorForAttachmentsOnly/BaseAnchorForAttachmentsOnly.js

```diff
@@ -27,28 +30,13 @@ const defaultProps = {
 };
 
 class BaseAnchorForAttachmentsOnly extends React.Component {
```

Comment:
Can we convert this to a functional component?

Reply:
Done